test-common/src/main/java/com/tc/util/Grep.java

```diff
@@ -1,22 +1,20 @@
 /*
+ *  Copyright Terracotta, Inc.
```

Comment:
Should have a look at https://github.com/Terracotta-OSS/dso/blob/main/common/src/test/java/com/tc/util/Grep.java

Reply:
going to just remove the class.  Not used